third_party/com.google.cloud.tools.eclipse.jdt.launching/epl-v10.html

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
```

Comment:
Eclipse normally distributes an `about.html` that links to the license at `http://www.eclipse.org/legal/epl-v10.html`.